docs/usage/cli/filter.rst

```diff
@@ -13,16 +13,24 @@ augur filter
     :prog: augur
     :path: filter
 
-How we subsample sequences in the zika-tutoral
-==============================================
+Guides
```

Comment:
I really like the addition of these top-level guides for filtering and subsampling. Do you imagine eventually moving this content to a top-level guide in the docs hierarchy or maybe just linking to this from the top-level?

Reply:
I like @joverlee521's [suggestion](https://github.com/nextstrain/augur/pull/1425#discussion_r1503177166) of putting it under `docs/guides/bioinformatics`.